modules/creating-a-machine-pool-cli.adoc

```diff
@@ -0,0 +1,106 @@
+====
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a ROSA cluster.
+
+.Procedure
+
+* To add a machine pool that does not use autoscaling:
+. Create the machine pool and define the instance type, compute node count, and node labels:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster-name> \
+                          --name=<machine_pool_id> \ <1>
+                          --replicas=<compute_node_count> \ <2>
+                          --instance-type=<instance_type> \ <3>
+                          --labels=<key>=<value>,<key>=<value> <4>
+----
+<1> Specifies the name of the machine pool. Replace `<machine_pool_id>` with the name of your machine pool.
+<2> Specifies the number of compute nodes to provision. If you deployed ROSA using a single availability zone, this defines the number of compute nodes to provision to the machine pool for the zone. If you deployed your cluster using multiple availability zones, this defines the number of compute nodes to provision in total across all zones and the count must be a multiple of 3. The `--replicas` argument is required when autoscaling is not configured.
+<3> Sets the instance type for the compute nodes in your machine pool. The instance type defines the vCPU and memory allocation for each compute node in the pool. Replace `<instance_type>` with an instance type. The default is `m5.xlarge`.
+<4> Defines the labels for the machine pool. Replace `<key>=<value>,<key>=<value>` with a comma-delimited list of key-value pairs, for example `--labels=key1=value1,key2=value2`.
++
+The following example creates a machine pool called `mymachinepool` that uses the `m5.xlarge` instance type and has 2 compute node replicas. The example also adds two workload-specific labels:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=mycluster --name=mymachinepool --replicas=2 --instance-type=m5.xlarge --labels=app=db,tier=backend
+----
++
+.Example output
+[source,terminal]
+----
+I: Machine pool 'mymachinepool' created successfully on cluster 'mycluster'
+I: To view all machine pools, run 'rosa list machinepools -c mycluster'
+----
+
+* To add a machine pool that uses autoscaling:
+. Create the machine pool and define the autoscaling configuration, instance type and node labels:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster-name> \
+                          --name=<machine_pool_id> \ <1>
+                          --enable-autoscaling \ <2>
+                          --min-replicas=<minimum_replica_count> --max-replicas=<maximum_replica_count> \ <3>
```

Comment:
For readability (gets cut off on the screen) can we move "--max-replicas" to the next line?

Reply:
Good suggestion, thanks. Resolved.